common/app/common/NewNavigation.scala

```diff
@@ -535,6 +533,18 @@ object NewNavigation {
       }
     }
 
+    def getActivePillar(page: Page) = {
+      val sectionOrTagId = getSectionOrTagId(page);
+      val activeSectionLink = SectionLinks.sectionLinks.find { item =>
+        item.pageId == sectionOrTagId
+      }
+      val activePillarName = activeSectionLink.map {
+        _.parentSection.name
+      }.getOrElse("")
+
+      (sectionOrTagId, activePillarName)
```

Comment:
Would it make sense to create a case class `Pillar` to carry these data, rather than using a tuple?